Backpack-SwiftUI/GraphicPromo/Classes/BPKGraphicPromo.swift

```diff
@@ -70,6 +70,29 @@ public struct BPKGraphicPromo: View {
         self.verticalAlignment = verticalAlignment
     }
     
+    public init(
+        headline: String,
+        image: Image,
+        variant: Variant = .onDark,
+        sponsorTitle: String,
+        partnerLogo: Image,
+        sponsoredAccessibilityLabel: String
+    ) {
+        self.headline = headline
+        self.kicker = ""
+        self.subheadline = nil
+        self.image = image
+        self.type = .button
```

Comment:
Should we support both types? Our v1 would only support button (is redirecting internally) but maybe there's another use case where it could be possible to have it open in a web browser?

Reply:
sure, let's add it as input 👍